### PR TITLE
AST-1465 - Related posts are showing the same posts on all posts

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
Related posts are showing the same posts on all posts- Random order by

### Types of changes

Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
Steps to reproduce -*
This is is occurring when we check the parent page from here - https://share.bsf.io/JruxErBp
When we uncheck the parent category and keep the child category it is working fine.
Also - There is some issue with Random query - https://brainstormforce.slack.com/archives/CDS7TQ75Z/p1623746646442500
Video - https://share.bsf.io/BluxkN6K

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/modules/related-posts/customizer/class-astra-related-posts-configs.php [hunk 1]
```diff
@@ -359,7 +359,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'date'          => __( 'Date', 'astra' ),
 					'title'         => __( 'Title', 'astra' ),
 					'post-order'    => __( 'Post Order', 'astra' ),
-					'random'        => __( 'Random', 'astra' ),
+					'rand'          => __( 'Random', 'astra' ),
 					'comment-count' => __( 'Comment Counts', 'astra' ),
```
Comment (imnavanath) [inline inc/modules/related-posts/customizer/class-astra-related-posts-configs.php:363]:
```suggestion
					'comment_count' => __( 'Comment Counts', 'astra' ),
```

File: inc/modules/related-posts/customizer/class-astra-related-posts-configs.php [hunk 2]
```diff
@@ -359,7 +359,7 @@ public function register_configuration( $configurations, $wp_customize ) {
 					'date'          => __( 'Date', 'astra' ),
 					'title'         => __( 'Title', 'astra' ),
 					'post-order'    => __( 'Post Order', 'astra' ),
```
Comment (imnavanath) [inline inc/modules/related-posts/customizer/class-astra-related-posts-configs.php:361]:
```suggestion
					'menu_order'    => __( 'Post Order', 'astra' ),
```

File: changelog.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 v3.8.0(Unreleased)
 - Improvement: Site Background will be applied as background CSS for Full-Width Contained/Stretched layouts instead of Content Background.
+- Fix: Fixed random option in order by releated posts.
```
Comment (imnavanath) [inline changelog.txt:3]:
```suggestion
- Fix: Random orderby attribute not working correctly for Related Posts.
```

File: inc/modules/related-posts/customizer/class-astra-related-posts-configs.php (hunk 1 above)
Comment (GauravKhupse) [reply inc/modules/related-posts/customizer/class-astra-related-posts-configs.php]: changes done!

File: inc/modules/related-posts/customizer/class-astra-related-posts-configs.php (hunk 2 above)
Comment (GauravKhupse) [reply inc/modules/related-posts/customizer/class-astra-related-posts-configs.php]: Is this need menu order, need to discuss?

Comment (imnavanath) [reply inc/modules/related-posts/customizer/class-astra-related-posts-configs.php]: post-order is not valid orderby param for WP_Query, that's why suggested to menu_order. Check once - https://developer.wordpress.org/reference/classes/wp_query/#order-orderby-parameters

**Review by imnavanath — APPROVED** [review]

File: changelog.txt [hunk 4]
```diff
@@ -1,6 +1,7 @@
 v3.8.0(Unreleased)
 Fix: W3 validation - Removed an element with the attribute tabindex for dropdown menu toggle.
 - Improvement: Site Background will be applied as background CSS for Full-Width Contained/Stretched layouts instead of Content Background.
+- Fix: Random orderby attribute not working correctly for Related Posts.
```
Comment (premanshup) [inline changelog.txt:4]:
```suggestion
- Fix: Related Posts - Post Query - Order by Random option not working as expected.
```

Comment (GauravKhupse) [reply changelog.txt]: Changes done!

**Review by premanshup — APPROVED** [review]

**Pull request merged**